orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/monitoreddeploy/EvaluateDeploymentHealthTaskSpec.groovy

```diff
@@ -198,6 +203,50 @@ class EvaluateDeploymentHealthTaskSpec extends Specification {
     false              | null             || ExecutionStatus.FAILED_CONTINUE
   }
 
+  def "should return status as RUNNING when Retrofit http error is thrown"() {
```

Comment:
Since the code changes are happening in MonitoredDeployBaseTask, is it possible to add additional coverage to MonitoredDeployBaseTaskTest?  Is there anything specific to EvaluateDeploymentHealthTaskSpec that's important? 
 As well, MonitoredDeployBaseTaskTest is java and I'd much rather add more java code than groovy.

Reply:
The EvaluateDeploymentHealthTask is one of the child class of MonitoredDeployBaseTask . 
The executeInternal() method is overridden in this child class , and the there is also a call to evaluateHealth API in this method.
This API call is what we are mocking in the test case. So, I think this test case is more specific to EvaluateDeploymentHealthTask. 
Moving this to MonitoredDeployBaseTaskTest would make it more complex , because it requires lots of mocking